common/lib/xmodule/xmodule/tests/xml/factories.py

```diff
@@ -134,6 +134,13 @@ def parent(self, _create, extracted, **_):
 
         extracted._xml_node.append(self._xml_node)  # pylint: disable=no-member, protected-access
         extracted.policy.update(self.policy)
+        self._xml_node.attrib.update(  # pylint: disable=no-member, protected-access
+            {
+                key: value
+                for key, value in extracted._xml_node.attrib.iteritems()  # pylint: disable=no-member, protected-access
+                if key in InheritanceMixin.fields.keys()
+            }
+        )
```

Comment:
I'm not 100% confident I need to be touching the code in this file, but this is what I had to do in order to get the `self_paced` value defined in `course = xml.CourseFactory.build(end=str(TOMORROW), self_paced=str(True))` to pass down through its children. Previously it was only present on the course root but no children, so the test was always assuming `False`.